docs/mp/config/01_introduction.adoc

```diff
@@ -21,15 +22,52 @@
 :toc-placement: preamble
 :spec-name: MicroProfile Config
 :description: {spec-name} support in Helidon MP
-:keywords: helidon, mp, microprofile, config
+:keywords: helidon, mp, microprofile, config, encryption, reference
+
 
 
 == About {spec-name}
+Helidon MP Config is an implementation of https://github.com/eclipse/microprofile-config/[Eclipse MicroProfile Config] that now includes Helidon-specific meta-configuration and APIs. You can configure your applications using MicroProfile's config properties file and APIs and extend the configuration system with Helidon-specific meta-configuration and APIs. You can also write and register custom ConfigSources.
+
+=== Helidon MicroProfile Config Features
+
+Helidon MicroProfile Config offers the following features:
+
+* *References* +
+You can use dollar, curly brackets, and reference to another key in a value to provide references and contruct the value when you use meta-configuration file for MP configuration. You can either reference the whole value or reference the value and add prefix or suffix, and the referenced value is replaced with the property.
+
+[source=java]
+.Example
+----
+
+providers:
+    - abac:
+      # Adds ABAC Provider - it does not require any configuration
+    - oidc:
+        client-id: "${ALIAS=security.properties.idcs-client-id}"
```

Comment:
This example is wrong - the correct reference is by `${}`.
This is a deprecated feature from another component.
To fix it, just remove the `ALIAS=` from each value.

Correct example:
```yaml
- oidc:
      client-id: "${security.properties.idcs-client-id}"
```